src/hooks/tradingView/useChartLines.tsx

```diff
@@ -240,11 +243,12 @@ export const useChartLines = ({
       const oldPrice = order.triggerPrice ?? order.price;
       const newPrice = orderLine.getPrice();
 
-      if (!isNewOrderPriceValid(order, newPrice)) {
-        // TODO: Add final copy with localization here
+      const priceError = getOrderModificationError(order, newPrice);
+      if (priceError) {
         notify({
-          title: 'Bad price!!!',
-          body: 'Dont cross the book price pls',
+          title: stringGetter({ key: priceError.title }),
+          body: priceError.body && stringGetter({ key: priceError.body }),
+          icon: <$WarningIcon iconName={IconName.Warning} />,
```

Comment:
it would be nice to have a standard set of icons so that i just have to pass in the icon name to the NotificationDisplayData